administration/src/cards/sendCardConfirmationMail.ts

```diff
@@ -0,0 +1,36 @@
+import { ApolloClient, ApolloError } from '@apollo/client'
+
+import getMessageFromApolloError from '../errors/getMessageFromApolloError'
+import {
+  SendCardCreationConfirmationMailDocument,
+  SendCardCreationConfirmationMailMutation,
+  SendCardCreationConfirmationMailMutationVariables,
+} from '../generated/graphql'
+
+export class SendCardConfirmationMailError extends Error {
+  constructor(message: string) {
+    super(message)
+    this.name = 'SendCardConfirmationMailError'
+  }
+}
+const sendCardConfirmationMail = async (
+  client: ApolloClient<object>,
+  project: string,
+  recipientAddress: string,
+  recipientName: string,
+  deepLink: string
+): Promise<void> => {
+  const result = await client.mutate<
```

Comment:
This might also throw. Maybe, we should wrap the error in SendCardConfirmationMail?
Also we should probably adjust the error dialog when this happens (in the useCardGenerator hook?)?

Reply:
The error dialog could be more general like `Etwas ist schief gegangen beim erstellen der Karte` in we want it more precise we have to check if `MailExtension` is provided.

Can you specify what you mean by wrap the error? Enclose the `client.mutate` with try catch?

Reply:
Hmm... maybe what you did is already enough. What happens when the backend throws `MailNotSentException`? The user may want to know which email address this exception belongs to 🤔

Reply:
hm i didn't know why i couldn't properly catch errors with this custom function but i adjusted it to use the standard mutation hook and now it is catched. added the error handling and toasts there. I set the error toast to timeout 0 and show the mail address, so the user has to close it manually but also has sufficient time to check the mail address. Not sure if thats fine for you